controllers/operands/operandHandler.go

```diff
@@ -59,16 +59,19 @@ func NewOperandHandler(client client.Client, scheme *runtime.Scheme, ci hcoutil.
 		newAAQHandler(client, scheme),
 	}
 
-	if ci.IsOpenshift() {
+	if ci.HasOpenshiftConsole() {
```

Comment:
not sure about this whole logic. @tiraboschi - WDYT?

Reply:
I don't think that the SSP operator (line 64) will work on vanially k8s just because the Openshift Console has been deployed there.